mods/cnc/rules/aircraft.yaml

```diff
@@ -2,6 +2,8 @@ TRAN:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 750
+    GivesExperience:
+        Experience: 750
```

Comment:
Keeping the traits separate, and these are placeholder numbers. 

They might want to be changed, I just put them as the same value as the cost, (except for the British spy)